packages/mgt-chat/src/statefulClient/StatefulGraphChatListClient.ts

```diff
@@ -599,28 +617,29 @@ detail: ${JSON.stringify(eventDetail)}`);
     }
   }
 
-  /**
-   * Update the state with given message either replacing an existing message matching on the id or adding to the list
-   *
-   * @private
-   * @param {(GraphChatMessage)} [message]
-   * @return {*}
-   * @memberof StatefulGraphChatListClient
-   */
-  private updateMessages(message?: GraphChatMessage) {
-    if (!message) return;
-    this.notifyStateChange((draft: GraphChatListClient) => {
-      const index = draft.messages.findIndex(m => m.messageId === message.messageId);
-      // this message is not already in thread so just add it
-      if (index === -1) {
-        // sort to ensure that messages are in the correct order should we get messages out of order
-        draft.messages = draft.messages.concat(message).sort(MessageCreatedComparator);
-      } else {
-        // replace the existing version of the message with the new one
-        draft.messages.splice(index, 1, message);
-      }
-    });
-  }
+  // /**
```

Comment:
Should this just be removed?

Reply:
There are some manipulation code here around messages, I am not sure if would be useful to keep a reference here for now? I can remove it if we think otherwise.

Reply:
If you think it is helpful, leave it and we can remove later.